packages/jdk11.rb

```diff
@@ -4,11 +4,11 @@
 class Jdk11 < Package
   description 'The Oracle JDK is a development environment for building applications, applets, and components using the Java programming language.'
   homepage 'https://www.oracle.com/java'
-  version '11.0.16.1'
+  version "#{@_ver = '11.0.16.1'}-1"
```

Comment:
I didn't know you could do this.  I might have some luck extracting the version in a script afterall.

Reply:
The `=` expression can return its value :) But `rubocop` doesn't like it anyways:
```
irb(main):005:0> a=1
=> 1
```

Reply:
Sounds like another tyrannical rubocop rule we can bypass?